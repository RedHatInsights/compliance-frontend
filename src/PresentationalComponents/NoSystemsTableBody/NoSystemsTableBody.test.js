import NoSystemsTableBody from './NoSystemsTableBody';

describe('NoSystemsTableBody', () => {
    it('expect to render without error', () => {
        const wrapper = shallow(
            <NoSystemsTableBody />
        );

        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
