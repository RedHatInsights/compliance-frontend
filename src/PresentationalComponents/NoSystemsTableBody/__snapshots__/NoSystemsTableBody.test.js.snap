// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoSystemsTableBody expect to render without error 1`] = `
<Bullseye>
  <EmptyState
    variant="full"
  >
    <Title
      headingLevel="h2"
      size="lg"
      style={
        Object {
          "fontWeight": "bold",
        }
      }
    >
      No systems on this policy
    </Title>
    <EmptyStateBody>
      <TextContent>
        <Text
          component="p"
        >
          Add systems to this policy from the systems page
        </Text>
        <Text
          style={
            Object {
              "color": "var(--pf-global--warning-color--100)",
            }
          }
        >
          <ExclamationTriangleIcon
            color="currentColor"
            noVerticalAlign={false}
            size="sm"
          />
           
          Policies without systems will not have reports.
        </Text>
      </TextContent>
    </EmptyStateBody>
  </EmptyState>
</Bullseye>
`;
